URA-695/vcf_making.sh

```
@@ -0,0 +1,31 @@
+    bcftools norm -f GRCh38_GIABv3_no_alt_analysis_set_maskedGRC_decoys_MAP2K3_KMT2C_KCNJ18_noChr.fa.gz -m -any --keep-sum AD - |
+    bcftools filter -i "FORMAT/DP > 99 && FORMAT/AF > 0.03" - -o sentieon-tnbam_comparison/"${sample}_filtered_v3.2.0.vcf"
+    done
+
+for vcf_file in sentieon-tnbam_v5.0.1/*.vcf.gz
+    do sample=$(basename "$vcf_file" .vcf.gz)
+    bedtools intersect -header -u -a "$vcf_file" -b coding_unrestricted_GRCh38_myeloid_5bp_flank_v2.1.0.bed |
+    bcftools norm -f GRCh38_GIABv3_no_alt_analysis_set_maskedGRC_decoys_MAP2K3_KMT2C_KCNJ18_noChr.fa.gz -m -any --keep-sum AD - |
+    bcftools filter -i "FORMAT/DP > 99 && FORMAT/AF > 0.03" - -o sentieon-tnbam_comparison/"${sample}_filtered_v5.0.1.vcf"
+    done
+
+for i in sentieon-tnbam_comparison/*.vcf; do bgzip $i; done
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)